.gitignore

```diff
@@ -2,3 +2,6 @@ vendor/
 cmd/commento/commento
 *.db
 .idea
+node_modules
+assets/js/*.min.js
+assets/style/*.min.css
```

Comment:
Could you fix your git/editor to append newlines at the end?

(Just a wild guess: are you using emacs by any chance? Someone else I knew had a similar problem)